app/bindings/api/v0/bindings/presentation_response.rb

```diff
@@ -46,6 +50,12 @@ def initialize(attributes = {})
         end
       end
 
+      if attributes.has_key?(:'ratings')
+        if (value = attributes[:'ratings']).is_a?(Array)
+          self.ratings = value
+        end
+      end
+
```

Comment:
This is some weird auto-generated ruby :(

Reply:
well it is at least checking that it is an array before storing it?

Reply:
Yeah, I mean it works but I'd have written it as just:
```ruby
value = attributes[:ratings])
self.ratings = value if value.is_a?(Array)
```